### ISSUE TITLE
Use ViewPropTypes from 'deprecated-react-native-prop-types'

### ISSUE THREAD

**Issue by danieldegrassi** [iou90/react-native-autoheight-webview#228]
I'm getting this warning:

ViewPropTypes will be removed from React Native. Migrate to ViewPropTypes exported from 'deprecated-react-native-prop-types'

**Comment by stevekuznetsov** [comment]
On my end, the version of RN is new enough that this causes a failure in Metro (?) and renders the app broken.

```
$ react-native --version
8.0.2
```

**Comment by iou90** [comment]
It should be fixed on v1.6.2. (https://github.com/iou90/react-native-autoheight-webview/releases/tag/v1.6.2)

**Comment by mauricekleine** [comment]
@iou90 @stevekuznetsov this change is breaking my app:

```
TypeError: undefined is not a function (near '..._deprecatedReactNativePropTypes.default.shape...')
```

It looks like `deprecated-react-native-prop-types` is not a drop-in replacement for `prop-types` - e.g. there's not `.shape` exported.

I had to downgrade to v1.6.1 to get my app to work again

**Comment by danieldegrassi** [comment]
Likewise.

**Comment by BS-FastwebIT** [comment]
same issue...

**Comment by MykhailoButkevych-PE** [comment]
Same issue

**Comment by stevekuznetsov** [comment]
Oops, sorry all - I think https://github.com/iou90/react-native-autoheight-webview/pull/235 fixes it, but would appreciate a check!

**Comment by iou90** [comment]
"deprecatedReactNativePropTypes.default.shape is not a function" should be fixed on v1.6.3. (https://github.com/iou90/react-native-autoheight-webview/releases/tag/v1.6.3)
@MykhailoButkevych-PE @BS-FastwebIT @danieldegrassi @mauricekleine
